components/style/properties/longhand/position.mako.rs

```diff
@@ -79,13 +79,13 @@ ${helpers.single_keyword("justify-content", "flex-start flex-end center space-be
                          products="servo",
                          animatable=False)}
 
+// https://drafts.csswg.org/css-flexbox-1/#propdef-align-items
 // FIXME(heycam): Disable align-items in geckolib since we don't support the Gecko initial value
```

Comment:
No, because gecko has "normal" as an initial value (but not a value that will be parsed). I don't know how this differs from the real initial value (stretch).

If you want to support this you'll have to implement this as a custom property (in geckolib mode only) that has a "normal" initial value which is not parsed.

@heycam can we remove this initial value from Gecko?